src/gsi/get_gefs_for_regional.f90

```diff
@@ -229,14 +230,17 @@ subroutine get_gefs_for_regional
   do n=1,200
      read(10,'(a)',err=20,end=40)filename 
   enddo
-40 n_ens=n-1
+40 n_ens_temp=n-1
+   n_ens_gfs=n_ens_temp
+write(6,*)'the number of ensemble members in the filelist is ',n_ens_temp
+write(6,*)'The n_ens_gfs is adjusted to ',n_ens_gfs
```

Comment:
Should lines 235 and 236 remain?  If they should, please indent both of these lines two spaces (these lines should align with line 240).

Reply:
@MichaelLueken-NOAA  Thank you !  I will take care of  your and Li's questions/suggesitons when I come back to the office next Wenseday.   Have a good weekend!

Reply:
@TingLei-daprediction Should the two writes here be kept?  They almost look like they are debug writes.  If they should remain, please indent these two lines by two spaces so that they proper align with line 238 below.